src/test/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactoryUnitTest.java

```diff
@@ -1396,6 +1380,43 @@ void testCreateFuncotations(final String expectedGeneName,
         }
     }
 
+    @Test(dataProvider = "provideForTestCreateGencodeFuncotationBuilderWithTrivialFieldsPopulated")
+    void testCreateGencodeFuncotationBuilderWithTrivialFieldsPopulated(final VariantContext variant,
+                                                                       final Allele altAllele,
+                                                                       final GencodeGtfGeneFeature gtfFeature,
+                                                                       final GencodeGtfTranscriptFeature transcript ) {
+
+        final GencodeFuncotationBuilder builder =
+                GencodeFuncotationFactory.createGencodeFuncotationBuilderWithTrivialFieldsPopulated(variant, altAllele,
+                        gtfFeature, transcript);
+        final GencodeFuncotation gf = builder.gencodeFuncotation;
+
+        // Ultra-trivial checks:
```

Comment:
Add assert that end of the funcotation is consistent with end of the VariantContext

Reply:
Fixed!